walletrpc/service.proto

```diff
@@ -86,6 +86,13 @@ message PingResponse {
     int64 exit = 2;
 }
 
+message Address {
+    string address = 1;
+}
+message Amount {
+    int64 zatoshis = 1;
```

Comment:
shouldn't this be just "zatoshi"

Reply:
+1

Reply:
or even just "balance" in case we later do decide to add in "received" and want to do less refactoring at that time. I think `zatoshi` is fine though because it is more descriptive.

Reply:
On the topic of the name of this field, we should probably follow the convention used in `zcashd` rpcs, which is `valueZat`.